test/services/CountriesSpec.scala

```diff
@@ -16,19 +16,33 @@
 
 package services
 
-import services.Countries.allCountries
+import org.scalatest.{Matchers, WordSpec}
+import services.Countries._
 import services.model.Country
-import unit.base.UnitSpec
 
-class CountriesSpec extends UnitSpec {
+class CountriesSpec extends WordSpec with Matchers {
```

Comment:
Why not unit ?

Reply:
I was experimenting with Matchers.  Have now discovered that you can replace 'should' with 'must' (didn't seem to work before) and will therefore revert to UnitSpec

Reply:
`Matchers -> shouldBe`, `MustMatchers -> mustBe`, but if you're talking about description:

`"A Controller" should/must` both, should and must is supported with WordSpec